### ISSUE TITLE
Isolated catalogue from sliced archive does not work as --reference

### ISSUE THREAD

**Issue by mjacksonhill** [Edrusb/DAR#64]
Hello-- not sure if this is a bug or if there is some usage option I have overlooked. I have tried to comb through the manual and Google but haven't found any solutions.

I have an archive in 76 parts produced with something like the following:

```
dar -Q \
--create /store/foo \
--min-digits 3 \
--fs-root /to_store/foo \
--compression=lz4::100k \
--slice 100G \
--multi-thread 16 \
--exclude-compression "*.tgz" -Z "*.gzip" -Z "*.gz" -Z "*.zip" -Z "*.bzip" -Z "*.bzip2" -Z "*.bz2" -Z '*.zst' -Z "*.rar" -Z "*.Z" -Z "*.z"

```

After creating the archive, I isolated the catalogue, which produces an archive with one slice.
```
dar --min-digits 2 -Q --multi-thread 16 --isolate /store/catalogs/foo -A /store/foo
```

I've identified one bug regarding the reporting of missing archives. When trying to extract files from slice 43, slice 42 is also required since it contains a parent directory. However, before dar prompts me to supply slice 42, it unexpectedly prompts me to supply slice 1... and then, after I have supplied slice 1, slice 42. If I quit out and re-submit the prompt with slice 42 available, but not slice 1, it runs perfectly. So slice 1 was never actually needed.

My main issue is that even with the isolated archive, I cannot get dar to extract my files without the last slice present. In other words, I can only get it to complete with `--ref` to the isolated catalogue while slices 42, 43, and 76 are in the same directory.

```
dar --extract /store/foo \
--go-into /some/parent/folder/ \
--include somefile.txt \
--ref /store/catalogs/foo \
--comparison-field=ignore-owner \
--flat \
--multi-thread 8
```

It was my understanding that relying on the isolated catalogue should mean that I do not need the last slice. Have I misunderstood, and the last slice is always required?

**Comment by Edrusb** [comment]
Hi, thanks for your feedback.

For the first problem you mention about slice 1 requested but not used, I'd be interested in knowing the command line you used to extract the data.

for your main problem please have a look at the [dar-support mailing-list archive](https://sourceforge.net/p/dar/mailman/dar-support/), there this aspect explained, and solutions has been addressed in the thread 'Huge archive and last slice"

https://sourceforge.net/p/dar/mailman/message/58712759/
https://sourceforge.net/p/dar/mailman/message/58713583/

Thanks to followup support request [there](https://sourceforge.net/p/dar/mailman/) if possible (subscribing to dar-support mailing-list), you'll have more people to interact and share experiences with.

However, you can followup here for bug behavior --- the first point you reported.

Thanks for your comprehension,
Denis

**Comment by mjacksonhill** [comment]
Understood! I came back to this today to try and recreate the issue so I could report the command line I used, but everything worked fine-- and didn't require the prior slice 42 either. My guess is I didn't notice I had included the `--sequential-read` flag in my prior efforts, since I was playing with that in an attempt to minimize the number of slices I had to restore.

Regardless, apologies for wasting your time!
